src/Stratis.Bitcoin.Features.SmartContracts/Wallet/SmartContractWalletController.cs

```diff
@@ -177,68 +176,81 @@ public IActionResult GetHistory(string walletName, string address)
                 // Wallet manager returns only 1 when an account name is specified.
                 AccountHistory accountHistory = accountsHistory.First();
 
-                List<FlatHistory> items = accountHistory.History.OrderByDescending(o => o.Transaction.CreationTime).Where(x=>x.Address.Address == address).ToList();
+                List<FlatHistory> items = accountHistory.History.OrderByDescending(o => o.Transaction.CreationTime).Where(x => x.Address.Address == address).ToList();
```

Comment:
I would always put the .Where first because then there's less items to order, but now I need to look into .NET internals!

Reply:
We can actually remove this ordering, I will do so.